src/runtime-rs/crates/resource/src/network/utils/mod.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+pub(crate) mod address;
+pub(crate) mod link;
+pub(crate) mod netns;
+
+pub(crate) fn parse_mac(s: &str) -> Option<hypervisor::Address> {
+    let v: Vec<_> = s.split(':').collect();
+    if v.len() != 6 {
+        return None;
+    }
+    let mut bytes = [0u8; 6];
+    for i in 0..6 {
+        bytes[i] = u8::from_str_radix(v[i], 16).ok()?;
+    }
+
+    Some(hypervisor::Address(bytes))
+}
+
+pub(crate) fn get_mac_addr(b: &[u8]) -> String {
+    assert!(b.len() >= 6);
```

Comment:
Could we change the return value to a Result or Option

Reply:
Thanks for your review, change to `Result<String>`